File: java/src/main/java/br/com/vps/consulting/b2b/management/partner/application/usecase/list/ListPartnersUseCase.java
package br.com.vps.consulting.b2b.management.partner.application.usecase.list;

import br.com.vps.consulting.b2b.management.shared.core.page.PageCustom;

public interface ListPartnersUseCase {
    PageCustom<PartnerListOutput> execute(ListPartnersInput input);
}

File: kotlin/src/main/kotlin/br/com/vps/consulting/b2b/management/partner/application/usecase/list/ListPartnersUseCase.kt
package br.com.vps.consulting.b2b.management.partner.application.usecase.list

import br.com.vps.consulting.b2b.management.shared.core.page.PageCustom

interface ListPartnersUseCase {
    fun execute(input: ListPartnersInput): PageCustom<ListPartnersOutput>
}
